com.chartboost.mediation/Documentation/setup/ad-adapters.md

```diff
@@ -11,59 +11,120 @@ The Chartboost Mediation Unity SDK package contains samples with optional depend
 Each Chartboost Mediation Unity SDK sample contains a .xml file with dependencies for the adapter available platforms (Android/iOS). In order for such dependencies to be included into the final build the [Google's External Dependency Manager](https://github.com/googlesamples/unity-jar-resolver) is needed.
 The `EDM` makes sure to download and package any dependencies with your build.
 
-## ChartboostMediationDependencies.xml
+## Chartboost Mediation Adapters Window
 
-The `ChartboostMediationDependencies.xml` can be found in the `Package Manager/Sample` as seen in the following screenshot.
+Since Chartboost Mediation 4.X Adapters are no longer released at the cadence as the Chartboost Mediation SDK, this means is possible to receive adapter updates in between SDK updates. 
 
-> **_NOTE:_** This is a mandatory dependency and must be imported for proper utilization of the Chartboost Mediation Unity SDK.
+Until Chartboost Mediation Unity SDK 4.1.0, adapters were addded through the UPM Samples capability; however, this limited the ability to provide adapter updates in between SDK updates. As such, from Chartboost Mediation Unity SDK 4.2.0, we have created a brand new Editor Window, that will allow users to fetch Ad Adapter updates on demand. *see screenshot below*
```

Comment:
```suggestion
As of Chartboost Mediation Unity SDK 4.1.0, adapters were added through the UPM Samples capability; however, this limited the ability to provide adapter updates in between SDK releases. As such, from Chartboost Mediation Unity SDK 4.2.0, we have created a brand new Editor Window. This will allow users to fetch Ad Adapter updates on demand. *see screenshot below*
```